buildroot-external/rootfs-overlay/usr/lib/systemd/system/haos-bt-cache.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Remove Bluetooth cache entries older than one week
+
+[Service]
+Type=oneshot
+ExecStart=/bin/sh -c 'find /var/lib/bluetooth/*/cache -mindepth 1 -type f -mtime +7  -exec rm {} \;'
```

Comment:
Does the modifed time get changed evertime that the cached file is used? Would atime be a more accurate way to determine if the cache is stale?

Reply:
Yes, I validated this in my testing: All devices which are nearby have updated modification timestamps. Should have mentioned that in the original PR.

Reply:
I have 3x BTHome Temperature BLE sensors. As I mentioned above my whole cache seems like it would be cleared. Is there a concern with removing active cached files?

Reply:
Uh wait, you are saying your installation doesn't update modification timestamps? :thinking:

Reply:
Yeah that is definitely not what I am seeing here:

```
# find /var/lib/bluetooth/*/cache -mindepth 1 -type f -mtime +7 | wc -l
118
# ls -1 /var/lib/bluetooth/*/cache | wc -l
134
# 
```

However, that being said, modification time could rely on an implementation detail of how the cache is managed currently. If a cache file has been accessed, it might be used, so we should use atime.

Using atime leads to only slightly different numbers here:

```
# find /var/lib/bluetooth/*/cache -mindepth 1 -type f -atime +7 | wc -l
116
# ls -1 /var/lib/bluetooth/*/cache | wc -l
134
```

Reply:
> I have 3x BTHome Temperature BLE sensors. As I mentioned above my whole cache seems like it would be cleared. Is there a concern with removing active cached files?

It would lead to unnecessarily read of the GATT services/attributes. Slows down that first read and uses a bit more battery, but since that would be only once a week it wouldn't be a big deal.

That said, I think the BTHome devices' GATT service/attributes are not cached currently, at least I don't see any entries in the cache files. 

But in any case, by checking access time we would even prevent that read.